### ISSUE TITLE
Personal Websites ?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#280]
from : https://github.com/Jwhiles/cv-site/issues/28#issuecomment-273056468
In case it gets "_lost_" ...

<hr />

@Jwhiles what @bradreeder said. 😉  👍 

# _expanding_ on it a bit ...

White links on a pink background ... 🤔 

![johnwhiles-cv-white-links-pink-background](https://cloud.githubusercontent.com/assets/194400/22013202/c7b3ac4c-dc8f-11e6-971d-661304c9e316.png)

## Background Reading 

+ https://www.viget.com/articles/color-contrast
+ http://graphicdesign.stackexchange.com/questions/15142/which-is-easier-on-the-eyes-dark-on-light-or-light-on-dark
+ http://uxmovement.com/content/when-to-use-white-text-on-a-dark-background/
+ https://www.nngroup.com/articles/text-over-images

## Great Example Personal CV Website

if you're looking for a _personal_ site of a F&C 4 Grad that gets _results_, 
look no further than @markwilliamfirth  http://markwilliamfirth.com 😍  `#textbook`

![mark-1](https://cloud.githubusercontent.com/assets/194400/22013223/e16056cc-dc8f-11e6-98f8-6f1c13645f21.png)

![mark-2](https://cloud.githubusercontent.com/assets/194400/22013227/e48540f6-dc8f-11e6-97be-0239e5c85c65.png)

![mark-3](https://cloud.githubusercontent.com/assets/194400/22013228/e856fe86-dc8f-11e6-87f8-6f63d4fb01c1.png)

![mark-4](https://cloud.githubusercontent.com/assets/194400/22013229/ea70e22c-dc8f-11e6-9fa1-2e99de5a6a62.png)

Mark might not be working as a "developer", 
but he certainly is _leveraging technology to solve real human challenges_
and has a _high positive impact_ on the world. ❤️  ✅ 
If anything he hasn't allowed himself to be "_pigeon-holed_" as a "Dev".

# Learning from the _Best_

If you're searching for a GitHub-based CV that tells your prospective employer 
**_everything_ they need to know** in ***less than a minute*** - _which, let's face it, is an exceeding **generous** amount of time for someone who's time is worth £3/minute which is the case for "Engineering Managers" at top tech firms_ - have a look at: 
+ @Neats29 (F&C 4): https://github.com/Neats29/CV 
and/or 
+ @heron2014 (F&C 5): https://github.com/heron2014/CV

# tl;dr

Here's the "***user story***" you are solving for:

> As a `tech lead` who's **time is _scarce_**
> (_20+ team members & multiple projects demanding my attention and 100 CVs to read..._),
> I want to be **impressed** by a candidate
> in the ***first 5 seconds***
> so that I **_know_ the person**
> is **not _wasting_ my time**.

## No Personal Website...?

I'm _lucky_ in that I don't _have_ a personal website, 
it's not an "_accident_" or "_laziness_" ...
(_they are a lot of work to **maintain** and more often than not just show **narcissism**. 
not sayin' that's the case for your site, just in general..._) 

> I discovered pretty early on that **personal websites** _only **matter** if they are **amazing**_.

I _often_ think: would I be _more_ successful with a slick personal site. 
usually conclude: probably, yes.
But then I think about all my peers who have sites 
and how most of them aren't _making_ anything particularly interesting...
and end up going back to focussing on my work...

> _[**Content**](https://github.com/dwyl/the-book/issues/69) is the best_ [***calling card***](https://youtu.be/qwOdU02SE0w).

_However_...

## Expert Mode

`Given` that you already have a personal website and [_domain_](https://www.techopedia.com/definition/22799/vanity-domain)

If you ever decide to "shoot for the stars" and by leveling up to "**expert mode**": 
look at what @jongold is doing on http://jon.gold 😮 
he still has a http://www.jon.gold/txt for SEO/"findability"
(_sometimes I think he has too much time! 
but then I remember he's just 
doing what he [loves](https://twitter.com/jongold/status/817459894789865474)
so isn't "watching the clock" 
when he's in his [element](https://youtu.be/3TAqSBMZDY8) ..._)
BTW: someone who _consistently_ prioritises [_getting out_](https://twitter.com/jongold/status/779333838195556352) [_of the building_](https://github.com/foundersandcoders/master-reference/issues/64) 
though not at the "_expense_" of his [_deep_](https://github.com/dwyl/hq/issues/169) technical knowledge/skill ... 😉 
Highly recommend reading: 
+ http://www.jon.gold/2015/07/functional-css/
+ http://www.jon.gold/2016/04/a-single-craft/
+ http://www.jon.gold/2016/06/declarative-design-tools/

> A personal website with _regularly updated content_ is 😍 
there's a _reason_ why @jongold is at the _top_ of his game, 
***so grateful*** we got to ["hangout" with (_and be **inspired** by_) him](https://www.meetup.com/Meteor-London/events/173719312/)  `before` he was ["_famous_"](https://twitter.com/jongold/status/755214622022569984) 
and know he's just a (_really_) _nice_ guy who _happens_ to be [***insanely talented***](https://twitter.com/jongold/status/804968786793091072)
and [works really hard](https://youtu.be/GtaxU6DZvLs?t=8s) when nobody is _watching_... 
Jon is [Living the "Tech Nomad" Dream](https://twitter.com/jongold/status/795517581235015680) ❤️ 
By contrast, @cwaring doesn't _have_ a "personal CV website" 
but he no less talented/successful than Jon... 🤔 

(_and even then I've seen other member of a hiring panel reject people who had "sick" personal websites because they hadn't been updated in ages or simply didn't show the skills they were claiming on their CV... but I digress..._)

## Conclusino

_Most_ of the time the "_Recruiter_" or "_Hiring Manager_" is only searching for keywords.
So having a good LinkedIn profile (_with the keywords you want to be "**found**" for_)
which then links to a GitHub profile which shows _deliberate practice_ 
is _more than enough_ for them to pick up the phone and call you.

> Your site is looking **_good_ so far**.
 But as a middle-class-white-male, 
you might want to _do_/_make_ something to _stand out_.

It's ***less*** about the ***words*** on the page (_or the colors for that matter_) 
and ***more*** about _demonstrating_ ***craft***.

**Comment by Jwhiles** [comment]
@nelsonic this is really helpful! Though you've destroyed my dream that simply using a monospace font would make every hiring manager love me. 

I appreciate that a personal site isn't the most important thing in the world, though we were advised by the speaker from Filament that having one (or at least a html CV) would in his eyes put us straight to the top of the pile.

My site uses markdown, so I'm planning to host an identical CV on github and then either link people to http://www.johnwhiles.com/#cv whilst hosting the exact same content on github. 

I'm struggling to think of ways of making myself _stand out_. I have significant voluntary experience, but it doesn't seem like something that would be particularly  relevant to a tech hiring manager - other than this I'm not sure what I could add?

I'm still in the process of revising my CV. If I send it over to you once I consider it done, would you be willing to give me some quick comments on the content?

**Comment by jsms90** [comment]
Amazingly useful set of comments for us all @nelsonic 

Really great advice:
> It's less about the words on the page (or the colors for that matter) and more about demonstrating craft.

But @Jwhiles, your use of colour and wording like:
> After graduating I worked as a fashion model. This broadened my horizons and ruined my chances of ever having my own website be the first Google result for my name.

are a huge part of communicating your personality :grin: Never change :wink:

**Comment by nelsonic** [comment]
@Jwhiles _always_ available for CV review/advice. 🤓 
There are quite a _few_ people in F&C ecosystem who have better _general_ CV experience than me. We need _general_ advice which people should follow as a _baseline_.
Then it's a question of building something interesting/useful that showcases your skills which you can use as a "portfolio"...

**Comment by jem-computer** [comment]
Miss you Nelson 💗

Some thoughts:
- **everything that provides content** should work without JavaScript. It's a pain but it's doable - jon.gold/Welcome etc etc are all server rendered, not just the blog.

- make sure it's easy to deploy & work on. I always skimp for personal sites because it seems like overkill at the time, but months later when you're trying to remember a complex series of commands on a server you haven't touched in a while you'll regret it. I guess this is a principle across projects, but I touch my personal sites much less than apps I work on.

**Comment by iteles** [comment]
@nelsonic We need to capture all this knowledge somewhere more public - it would be a shame to have this huge amount of info and Jon's comments get lost in an issue! :heart:

**Comment by nelsonic** [comment]
@iteles _nah_... no _need_ to "_capture_" this anywhere other than here, 
if people are interested they can read it ... 

I **100% acknowledge** that it's "_anecdotal_" and "_subjective_". 💭 
`.each` person should figure out (_for themselves_) if a personal website "_works_" for them.

I've worked with hiring/team/HR managers who _prioritise_ personal sites 
on a few occasions during _initial_ selection process; 
it _can_ make a difference _especially_ if the site is ***outstanding***.

Equally I've seen people **_rejected_ outright** because 
their personal site was stale, _unattractive_ or had ***invalid html***.
e.g: https://validator.w3.org/nu/?doc=http%3A%2F%2Fwww.johnwhiles.com
![image](https://cloud.githubusercontent.com/assets/194400/22124659/a5e7ab18-de88-11e6-8144-ff7c111e1bb3.png)
... not trying to "_pick on_" John just an example ...
by contrast: https://validator.w3.org/nu/?doc=http%3A%2F%2Fwww.dwyl.io
![image](https://cloud.githubusercontent.com/assets/194400/22124895/8a2f3746-de89-11e6-8211-fe1ef9e79b4e.png)


**Point** is: `.some` people pay (_way_) more attention to this than others.
I would _definately_ (_always_) look at a personal site and _factor_ it in selection process, 
but would hope to avoid _penalising_ someone 
because they didn't have _time_ to keep a personal site up-to-date.
I pay _way_ more attention to knowledge sharing and GitHub activity.
_specifically_ work on _both_ personal ***projects*** (_are the projects "hello world" or **interesting**..._?)
and contribution to Open Source projects they _**use** but **don't** own_! e.g: https://github.com/rtfeldman/elm-workshop/issues/8
(_this is the **best proxy** for judging if the person going to **prioritise** the **community** over themselves?_)

**Comment by Jwhiles** [comment]
![image](https://cloud.githubusercontent.com/assets/20307225/22129720/a8e2e9e2-de9f-11e6-8524-f98d96ffbb0d.png)
Thanks @nelsonic !

**Comment by Jwhiles** [comment]
That's a really ridiculously incredibly useful website that I'm horrified I've never seen before

**Comment by jsms90** [comment]
@nelsonic I agree with @iteles - [this needs to be captured](https://github.com/foundersandcoders/master-reference/issues/71)

@Jwhiles Unless I'm missing something, you don't need a whole website to do this. The accessibility and HTML outliner chrome extensions are good for this :+1:

**Comment by nelsonic** [comment]
https://zen-of-programming.com/kickass-portfolio/
Via/comments: https://news.ycombinator.com/item?id=17671490

**Comment by nelsonic** [comment]
Tempted by a `.dev` domain ... reasonably priced (_$12 per annum_) https://domains.google/tld/dev/
Google is giving ["Domainers"](https://en.wikipedia.org/wiki/Domain_name_speculation) and Companies a chance to snap them up for a premium ...
![image](https://user-images.githubusercontent.com/194400/52901517-8e464400-31fc-11e9-81ab-36875435aa12.png)

**Comment by jem-computer** [comment]
What a wretched game of chicken to play with your credit card! I'm looking
for a snappy domain for a new thing (that I might DM you about before it's
public, Nelson) and this looks perfect. Now, to calculate the imagined
popularity of the 6-letter noun I want. What day do I buy it on?

**Comment by nelsonic** [comment]
https://writing.markchristian.org/2019/04/29/personal-web-sites
![image](https://user-images.githubusercontent.com/194400/56963045-ea5c0b80-6b4f-11e9-9040-a4108ead0b01.png)
https://news.ycombinator.com/item?id=19784907

**Comment by nelsonic** [comment]
https://www.dannysalzman.com/2020/04/08/analyzing-hn-readers-personal-blogs

**Comment by nelsonic** [comment]
Good example of GitHub Profile page: https://github.com/doctopus
<img width="852" alt="image" src="https://github.com/nelsonic/nelsonic.github.io/assets/194400/c6f0cc99-fe5a-4a9c-876b-b472487f0852">

Via: https://github.com/anuraghazra/github-readme-stats
#NoteToSelf 

e.g: 
```md
<!-- Top Languages & Streaks-->
[![Doctopus's GitHub Top Languages-Dark](https://github-readme-stats.vercel.app/api/top-langs?username=doctopus&show_icons=true&locale=en&layout=compact&hide_border=true&theme=dark#gh-dark-mode-only)](https://github.com/doctopus#gh-dark-mode-only)
[![Doctopus's GitHub Top Languages-Light](https://github-readme-stats.vercel.app/api/top-langs?username=doctopus&show_icons=true&locale=en&layout=compact&hide_border=true&theme=default#gh-light-mode-only)](https://github.com/doctopus#gh-light-mode-only)
[![Doctopus's Github Streaks -Dark](https://github-readme-streak-stats.herokuapp.com?user=doctopus&hide_border=true&date_format=M%20j%5B%2C%20Y%5D&card_width=490&theme=dark#gh-dark-mode-only)](https://github.com/doctopus#gh-dark-mode-only)
[![Doctopus's Github Streaks -Light](https://github-readme-streak-stats.herokuapp.com?user=doctopus&hide_border=true&date_format=M%20j%5B%2C%20Y%5D&card_width=490&theme=light#gh-light-mode-only)](https://github.com/doctopus#gh-light-mode-only)
```


[![Doctopus's GitHub Top Languages-Dark](https://github-readme-stats.vercel.app/api/top-langs?username=nelsonic&show_icons=true&locale=en&layout=compact&hide_border=true&theme=dark#gh-dark-mode-only)](https://github.com/doctopus#gh-dark-mode-only)
[![Doctopus's GitHub Top Languages-Light](https://github-readme-stats.vercel.app/api/top-langs?username=nelsonic&show_icons=true&locale=en&layout=compact&hide_border=true&theme=default#gh-light-mode-only)](https://github.com/doctopus#gh-light-mode-only)
[![Doctopus's Github Streaks -Dark](https://github-readme-streak-stats.herokuapp.com?user=nelsonic&hide_border=true&date_format=M%20j%5B%2C%20Y%5D&card_width=490&theme=dark#gh-dark-mode-only)](https://github.com/doctopus#gh-dark-mode-only)
[![Doctopus's Github Streaks -Light](https://github-readme-streak-stats.herokuapp.com?user=nelsonic&hide_border=true&date_format=M%20j%5B%2C%20Y%5D&card_width=490&theme=light#gh-light-mode-only)](https://github.com/doctopus#gh-light-mode-only)
